src/installer/managed/Microsoft.NET.HostModel/MachO/LinkEditCommand.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Runtime.InteropServices;
+
+namespace Microsoft.NET.HostModel.MachO;
+
+[StructLayout(LayoutKind.Sequential)]
+internal struct LinkEditCommand
+{
+    private readonly MachLoadCommandType _command;
+    private readonly uint _commandSize;
+    private readonly uint _dataOffset;
+    private readonly uint _dataSize;
+
+    public LinkEditCommand(MachLoadCommandType command, uint dataOffset, uint dataSize, MachHeader header)
+    {
+        _command = (MachLoadCommandType)header.ConvertValue((uint)command);
+        _commandSize = header.ConvertValue((uint)Marshal.SizeOf<LinkEditCommand>());
+        _dataOffset = header.ConvertValue(dataOffset);
+        _dataSize = header.ConvertValue(dataSize);
+    }
+
+    public bool IsDefault => this.Equals(default(LinkEditCommand));
+
+    internal uint GetDataOffset(MachHeader header) => header.ConvertValue(_dataOffset);
```

Comment:
Is this used anywhere? Can it be moved into the testing library?

Reply:
Yes, we use this when removing a signature, and determining how much of the file needs to be hashed (the signature blob is not included in the code hashes).